src/atomate2/common/jobs/anharmonicity.py

```diff
@@ -0,0 +1,611 @@
+"""Jobs for running anharmonicity quantification."""
+
+from __future__ import annotations
+
+import contextlib
+import logging
+from typing import TYPE_CHECKING
+
+import numpy as np
+from jobflow import Flow, Response, job
+from phonopy import Phonopy
+from pymatgen.core import Structure
+from pymatgen.core.units import kb
+from pymatgen.io.phonopy import get_phonopy_structure, get_pmg_structure
+
+from atomate2.aims.schemas.calculation import Calculation
+from atomate2.common.schemas.anharmonicity import AnharmonicityDoc
+
+if TYPE_CHECKING:
+    from pathlib import Path
+    from typing import Any
+
+    from atomate2.aims.jobs.base import BaseAimsMaker
+    from atomate2.common.schemas.phonons import ForceConstants, PhononBSDOSDoc
+    from atomate2.forcefields.jobs import ForceFieldStaticMaker
+    from atomate2.vasp.jobs.base import BaseVaspMaker
+
+
+logger = logging.getLogger(__name__)
+
+# Constants
+EV = 1.60217733e-19  # [J]
+AA = 1e-10  # [m]
+AMU = 1.66053904e-27  # [kg]
+THZ = 1e12  # [1/s]
+PI = np.pi
+OMEGA_TO_THZ = (EV / AA**2 / AMU) ** 0.5 / THZ / 2 / PI  # 15.633302 THz
```

Comment:
can't it be imported from pymatgen or elsewhere? Especially eV to J or amu to kg?

Reply:
https://github.com/materialsproject/pymatgen/blob/master/src/pymatgen/core/units.py (see here)

Reply:
I suggested this initially because I was not sure which CODATA was used for different codes. The impact should be minor, but I know VASP and aims have slightly different definitions of the electronvolt because of it.

Reply:
```
from pymatgen.core.units import Energy, Mass, Length, FloatWithUnit
import numpy as np 

inner = Energy(1, "eV") / (Length(1, "ang"))**2 / Mass(1, "amu")
inner_sqrt = np.sqrt(inner.unit.as_base_units[1])
omega_to_THz = inner_sqrt / FloatWithUnit(1, "THz").to("Hz") / 2 / np.pi
```
I did some messing around with the pymatgen units and found a way that works and gets to the same OMEGA_TO_THZ conversion factor as is currently in the code

Reply:
Ah, i see. If this is required for aims, this is absolutely fine, of course.

Reply:
Or maybe you define the aims units once and then import them? I think there have been unit conversions elsewhere in the code, right?

Reply:
Ah, here they are: https://github.com/materialsproject/atomate2/blob/main/src/atomate2/aims/utils/units.py

I think you might want to reuse these definitions

Reply:
Ya we should use these. I forgot to check that.

Reply:
I just changed the code to import the conversion factor from atomate2 rather than re-adding it to the anharmonicity workflow